### DISCUSSION TITLE
motion journalctl duplicated entries

### DISCUSSION THREAD

**Discussion by eusoubrasileiro** [Motion-Project/motion#1608] [Q&A]
Hi everyone, thanks again for this awesome project.
It's a stupid question but there we go since I'm struggling with this for a long time.

I am using `log_level 6` and executing motion with `motion -n`

How to avoid having duplicated log entries on journalctl? Example from motion of undesired behavior bellow.

`when I use journalctl -xe -u motion`

> Dec 18 14:33:11 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:33:11 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:33:11 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_frontwall/12_18/202212181433113.mp4
> Dec 18 14:33:11 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_frontwall/12_18/202212181433113.mp4
> Dec 18 14:33:38 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [ALL] motion_detected: Motion detected - starting event 33.
> Dec 18 14:33:38 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [ALL] motion_detected: Motion detected - starting event 33.
> Dec 18 14:33:38 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:33:38 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:33:38 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_street/12_18/202212181433384.mp4
> Dec 18 14:33:38 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_street/12_18/202212181433384.mp4
> Dec 18 14:33:58 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_kitchen/12_18/202212181430131.jpg
> Dec 18 14:33:58 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_kitchen/12_18/202212181430131.jpg
> Dec 18 14:33:58 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] mlp_actions: End of event 19
> Dec 18 14:33:58 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] mlp_actions: End of event 19
> Dec 18 14:34:14 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_frontwall/12_18/202212181433113.jpg
> Dec 18 14:34:14 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_frontwall/12_18/202212181433113.jpg
> Dec 18 14:34:14 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ALL] mlp_actions: End of event 21
> Dec 18 14:34:14 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ALL] mlp_actions: End of event 21
> Dec 18 14:34:28 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ALL] motion_detected: Motion detected - starting event 22.
> Dec 18 14:34:28 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ALL] motion_detected: Motion detected - starting event 22.
> Dec 18 14:34:28 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:34:28 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:34:28 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_frontwall/12_18/202212181434283.mp4
> Dec 18 14:34:28 jetson-nano motiond[5819]: [3:ml3:frontwall] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_frontwall/12_18/202212181434283.mp4
> Dec 18 14:36:20 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_street/12_18/202212181433384.jpg
> Dec 18 14:36:20 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_street/12_18/202212181433384.jpg
> Dec 18 14:36:20 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [ALL] mlp_actions: End of event 33
> Dec 18 14:36:20 jetson-nano motiond[5819]: [4:ml4:street] [NTC] [ALL] mlp_actions: End of event 33
> Dec 18 14:41:16 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] motion_detected: Motion detected - starting event 20.
> Dec 18 14:41:16 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] motion_detected: Motion detected - starting event 20.
> Dec 18 14:41:16 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:41:16 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:41:16 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_kitchen/12_18/202212181441161.mp4
> Dec 18 14:41:16 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_kitchen/12_18/202212181441161.mp4
> Dec 18 14:42:19 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_kitchen/12_18/202212181441161.jpg
> Dec 18 14:42:19 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing image to file: /mnt/motion/pictures/cam_kitchen/12_18/202212181441161.jpg
> Dec 18 14:42:19 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] mlp_actions: End of event 20
> Dec 18 14:42:19 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] mlp_actions: End of event 20
> Dec 18 14:42:32 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] motion_detected: Motion detected - starting event 21.
> Dec 18 14:42:32 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ALL] motion_detected: Motion detected - starting event 21.
> Dec 18 14:42:32 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:42:32 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [ENC] ffmpeg_passthru_codec: pass-through mode enabled.  Changing to MP4 container.
> Dec 18 14:42:32 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_kitchen/12_18/202212181442321.mp4
> Dec 18 14:42:32 jetson-nano motiond[5819]: [1:ml1:kitchen] [NTC] [EVT] event_newfile: Writing movie to file: /mnt/motion/movies/cam_kitchen/12_18/202212181442321.mp4
> 

I suppose is something wrong with my motion settings is making duplicated entries on journalctl. 
Any idea I appreciate.

**Comment by tosiara** [comment]
Could you try to run motion with logging into file enabled? Does it also duplicate entries when logging into a file?
